setup.py

```diff
@@ -24,7 +24,7 @@ def find_version(*file_paths):
 
 
 requires = ['jmespath>=0.7.1,<1.0.0',
-            'python-dateutil>=2.1,<3.0.0',
+            'python-dateutil>=2.1,<2.7.0',
```

Comment:
Would it make sense to use different version ranges for Python 2.6 and every other version of Python? That'd limit the potential missing out of bugfixes and such only to people using 2.6.

Reply:
:+1:
https://github.com/boto/botocore/pull/1406

freezegun has this btw: https://github.com/spulec/freezegun/blob/6bb258967769aa1117c00affc3a4bdab5db0ddee/setup.py#L8-L12J